test/service_test.js

```diff
@@ -48,7 +50,7 @@ suite('service', () => {
         // Close the request to /log after some time elapses.
         setTimeout(async function() {
           // At this point we should have a logcat process.
-          let [ps] = await exec('ps -au');
+          let [ps] = await exec('ps au');
```

Comment:
This command didn't work on my Mac with the dash `-`, so not sure if this breaks Linux yet.